domain/src/main/java/com/example/domain/usecase/GetHeartRateUseCaseImpl.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.domain.usecase
+
+import com.example.domain.model.HeartRateInfo
+import com.example.domain.model.Result
+import com.example.domain.repository.HeartRateMeasureRepository
+
+class GetHeartRateUseCaseImpl(private val heartRateMeasureRepository: HeartRateMeasureRepository):GetHeartRateUseCase {
```

Comment:
Please rename interface class to start with I so you don't have to write imp at the end of class name every time

Reply:
Renamed